charts/solr/templates/pvc-solr-configs.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: solr-configs
+spec:
+  accessModes:
+    - ReadWriteMany
```

Comment:
Not compatible with AWS.  Only ReadWriteOnce is supported.

Reply:
Really I should say "ReadWriteMany" is only supported by a subset of Kubernetes persistent volume drivers -  (6 out of 19 always with a few others depending on the drive) https://kubernetes.io/docs/concepts/storage/persistent-volumes/

Reply:
So...had another thought that I might explore which is building a docker container from the official solr docker container, that has the sunspot stuff baked in so it just works.